components/raftstore/src/store/fsm/peer.rs

```diff
@@ -999,10 +1007,9 @@ where
                 if self.fsm.peer.raft_group.raft.leader_id != raft::INVALID_ID
                     // the returned region is stale
                     || util::is_epoch_stale(
-                        region.get_region_epoch(),
-                        self.fsm.peer.region().get_region_epoch(),
-                    )
-                {
+                    region.get_region_epoch(),
+                    self.fsm.peer.region().get_region_epoch(),
+                ) {
```

Comment:
Is this an unexpected format change?

Reply:
Fixed~ BTW, I would like to ask how to specify code style during the development of TiKV? IoTDB, for example, uses the [Google Code Format specification](https://github.com/apache/iotdb/blob/master/java-google-style.xml) to avoid such problems. These code format changes are all caused by my use of the format code in Clion and `Cargo format`. I would like to know how you avoid these problems?

Reply:
I think `Cargo fmt` is enough, @sticnarf are there any other requirements?

Reply:
Just `cargo fmt` is enough. I think the unexpected format change is because you use Clion to format your code. Don't let Clion format your code, or let Clion use rustfmt to format.

Reply:
Got it! Thanks!